src/pages/admin/downloads.tsx

```diff
@@ -0,0 +1,7 @@
+import Downloads from "pageLayouts/Downloads";
+
+const DownloadPage = () => {
```

Comment:
I am guessing you added these `tsx` files but did not delete the `jsx` right? If that is the case, feel free to  delete the `jsx` in this PR